reference/build_helpers/meson.rst

```diff
@@ -35,3 +35,71 @@ Methods
     - **args**: A list of additional arguments to be passed to the ``make`` command. Each argument will be escaped according to the current shell. No extra arguments will be added if ``args=None``
     - **build_dir**: Default conanfile.build_folder
     - **targets**: A list of targets to be built.
```

Comment:
In the example at the top of the page, change the `meson.configure()` with a `meson.configure(build_dir="%s/build" % self.build_folder)`